src/main/java/com/ksabbak/javaserver/Routes.java

```diff
@@ -0,0 +1,41 @@
+package com.ksabbak.javaserver;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Routes {
+    private static final Map<String, String[]> routes = initMap();
+
+    private static Map<String, String[]> initMap(){
+        Map<String, String[]> map = new HashMap<String, String[]>();
+        map.put("/", new String[] {"GET", "HEAD"});
+        map.put("/logs", new String[] {"GET"});
+        map.put("/requests", new String[] {"HEAD"});
+        map.put("/these", new String[] {"PUT"});
+        map.put("/cookie", new String[] {"GET"});
+        map.put("/eat-cookie", new String[] {"GET"});
+        map.put("/cat-form", new String[] {"POST"});
+        map.put("/coffee", new String[] {"GET"});
+        map.put("/tea", new String[] {"GET"});
+        map.put("/redirect", new String[] {"GET"});
+        map.put("/method_options", new String[] {"GET", "PUT", "POST", "HEAD", "OPTIONS"});
+        map.put("/method_options2", new String[] {"GET", "OPTIONS"});
+        map.put("/form", new String[] {"POST"});
+        map.put("/put-target", new String[] {"PUT"});
+        map.put("/file1", new String[] {"GET"});
+        map.put("/text-file.txt", new String[] {"GET"});
+
+        return Collections.unmodifiableMap(map);
```

Comment:
I think this is an area where you might be jumping ahead a bit. Maybe stick to the routes that you're already covering in your tests and then add these back in as you get to the rest? I suspect this data structure shape will end up changing also when you end up finding that you want to return different responses for different methods on the same route.

Reply:
Removed!